### ISSUE TITLE
Dependencies missing from npm package.json

### ISSUE THREAD

**Issue by yvon** [ConsenSys-archive/drizzle-legacy#59]
https://github.com/trufflesuite/drizzle/blob/master/dist/package.json

Drizzle dependencies are not defined, I had to manually update my project web3 version to match drizzle requirements and add: 

```diff
+    "eth-block-tracker-es5": "^2.3.2",
+    "redux": "^4.0.0",
+    "redux-saga": "^0.16.0",
-    "web3": "^0.20.6",
+    "web3": "1.0.0-beta.34",
```

**Comment by yvon** [comment]
I pushed a commit to my fork, https://github.com/yvon/drizzle/commit/6663ff2f96ced1cf1c83ba94ad9af23152682ef3, and I was about to submit a PR, but I found another approach.

I don't understand why the Npm package is generated from dist folder. From my POV, dist compressed version is useful only for people who would like to download/include the script without Npm.

If I'm right:
+ npm package should include all sources
+ ~~dist version should embed dependencies~~

I ended up installing the package from github url:

```diff
-    "drizzle": "^1.1.5",
+    "drizzle": "trufflesuite/drizzle",
```

**Comment by sliptype** [comment]
@yvon I think you should still make that PR. After looking at a few other packages, it looks to me like the standard practice is to define the dependencies in `dist/package.json` and not embed them in the compiled file (`dist/drizzle.js`).

Right now, anyone that installs drizzle does not have the correct dependencies.

Source maps are enabled in `webpack.config.js` so the development workflow should look like this:

1. `git clone https://github.com/trufflesuite/drizzle.git`
2. `cd drizzle && yarn install`
3. `cd dist && yarn link`
4. `cd ../../my-application && yarn link drizzle`
5. `cd ../drizzle && yarn dev`

Now as you make changes, drizzle will be compiled and updated in `my-application` and you can debug properly using the source maps!

**Comment by JFGHT** [comment]
Related to:

https://github.com/trufflesuite/drizzle/issues/53 and https://github.com/trufflesuite/drizzle/issues/58

**Comment by yvon** [comment]
@sklingler93 

> After looking at a few other packages, it looks to me like the standard practice is to define the dependencies in dist/package.json and not embed them in the compiled file (dist/drizzle.js).

You are right. I'm new to node. I'll update my comment.

> I think you should still make that PR.

But no, I believe they should

+ generate an npm package with all sources, including the dist folder.
+ get ride of dist/package.json and use the root one

Actually exactly how it behaves when we install the package from github url.

**Comment by sliptype** [comment]
@yvon You're right, that sounds like the best approach

**Comment by JFGHT** [comment]
I also managed to fix this issue by installing directly from github url but in Windows it gives error.

Using the Windows 10's Ubuntu and installing from that terminal gives no error at all.

**Comment by JFGHT** [comment]
@DiscRiskandBisque This issue should be addressed.

**Comment by DiscRisknBisque** [comment]
This should now be fixed in v1.2.2. You've corrected hit the issue--the dependencies weren't added to the release `pacakge.json` so were not installing unless otherwise present in the project already.
